src/components/User/CUserEditorPassword.vue

```diff
@@ -144,6 +159,9 @@ export default {
       this.password = ''
       this.confirmPassword = ''
     },
+    onRemovePassword () {
+      this.$emit('submit', '')
```

Comment:
There is no need to emit anything if you don't need to emit anything.

`this.$emit('submit')` should work just fine.
Further, you can move the emit call into the template as there is no need for another oneliner function (make sure to remove `this.` if you decide to do so).

Reply:
@MarioBur please clean this one up as Tomaz's suggested.

`@confirmed="$emit('submit')"` should suffice